src/backends/tensorrt/plan_backend.cc

```diff
@@ -356,19 +412,46 @@ PlanBackend::Context::ValidateOutputs(
   return Status::Success;
 }
 
+Status
+PlanBackend::Context::NormalizeDims(
```

Comment:
If Normalize the correct term. Aren't you finding the maximum tensor shape? Perhaps MaximumDims?